core/src/main/java/org/opensearch/sql/expression/function/FunctionDSL.java

```diff
@@ -323,13 +408,29 @@ public SerializableTriFunction<ExprValue, ExprValue, ExprValue, ExprValue> nullM
     };
   }
 
+  /**
+   * Wrapper the varargs ExprValue function with default NULL and MISSING handling.
+   */
+  public SerializableVarargsFunction<ExprValue, ExprValue> nullMissingHandling(
+          SerializableVarargsFunction<ExprValue, ExprValue> function, boolean withVarargs) {
+    return (args) -> {
+      if (Arrays.stream(args).anyMatch(ExprValue::isMissing)) {
+        return ExprValueUtils.missingValue();
+      } else if (Arrays.stream(args).anyMatch(ExprValue::isNull)) {
```

Comment:
`else` not needed here

Reply:
Addressed in bed81bfefc360c00ed64e9609ceb3f02f2513fc9